### PR TITLE
asset/manifests/gcp: update the config to inlude master and worker node-tags

### PR DESCRIPTION
**Pull request by abhinavdahiya**
Tags for both master and worker need to be set, because in 4.3 both master and control-plane can be backends for k8s service type Load balancer and these are used
to create the correct firewal rules.

The cuurrent impl uses `gopkg.in/ini.v1` for writing the configuration, but the gce k8s code uses `gopkg.in/gcfg.v1` to read the configuration.
Although both are similar, gcfg handles multi-valued differently [1] which is esp problamatic for node-tags.

gcfg multi-value
```
node-tags = a
node-tags = b
```

ini multi-value
```
node-tags = [a, b]
```

Also gcfg only implements a reader and no Marshaller [2], therefore for the purpose of installer which is onl supposed to control few thing using go-templates to perform
the operation of marshalling seems acceptable.

[1]: https://github.com/go-gcfg/gcfg/blob/61b2c08bc8f6068f7c5ca684372f9a6cb1c45ebe/doc.go#L93-L96
[2]: https://github.com/go-gcfg/gcfg/blob/61b2c08bc8f6068f7c5ca684372f9a6cb1c45ebe/doc.go#L140

/cc @smarterclayton @jstuever

### CODE REVIEW

**Comment by jstuever** [comment]
looks good to me.

File: pkg/asset/manifests/gcp/cloudproviderconfig.go [hunk 1]
```diff
@@ -36,18 +33,27 @@ func CloudProviderConfig(infraID, projectID, subnet string) (string, error) {
 			Multizone: true,
 
 			// To make sure k8s cloud provide has tags for firewal for load balancer.
```
Comment (wking) [inline pkg/asset/manifests/gcp/cloudproviderconfig.go]: drive-by nit: `firewal` -> `firewall`.  Don't care if this gets fixed ;)
